src/data.yaml

```diff
@@ -2947,6 +2947,34 @@ jobs:
         long: 
     review: 
     rating: 
+  - name: B Lab Global
+    website: http://www.bcorporation.net/
+    jobs: https://boards.greenhouse.io/blab?gh_jid=5505108002
```

Comment:
They don't list any dev jobs here. So I'd rather not list B Lab Global.

Reply:
no problem - I have seen dev jobs on there but none at the moment as you say